src/Command/User/AddUserCommand.php

```diff
@@ -80,7 +81,8 @@ protected function configure(): void
             ->addArgument('email', InputArgument::OPTIONAL, 'The email of the new user')
             ->addArgument('password', InputArgument::OPTIONAL, 'The plain password of the new user')
             ->addArgument('full-name', InputArgument::OPTIONAL, 'The full name of the new user')
-            ->addOption('admin', null, InputOption::VALUE_NONE, 'If set, the user is created as an administrator')
+            ->addArgument('role', InputArgument::OPTIONAL, 'The role of the user [editor|admin]')
+            ->addArgument('tenant-keys', InputArgument::IS_ARRAY | InputArgument::OPTIONAL, 'The keys of the tenants the user should belong to (separate multiple keys with a space)')
```

Comment:
I would use options for this (rather than arguments) to make it easier to handle multiple values, e.g.

```sh
app:user:add … --role=admin --tenant=hat --tenant=briller
```

but that's a matter of opinion.

Reply:
And that example uses an option, `--admin`, for (implicitly) setting the role. This is not really important, so let's leave it as it is for now.

Reply:
Agreed 😄 

Changed because 
1) there might be more than 2 roles at some point, 
2) none of the people who has had to use the command have figured out the `--admin` option. So tried to make it more obvious how to select role